new-lamassu-admin/src/components/machineActions/MachineActions.styles.js

```diff
@@ -0,0 +1,79 @@
+import { fade } from '@material-ui/core/styles/colorManipulator'
```

Comment:
Moving the styles here didn't leave a bunch of orphan styles at the machine components compontent?